Gruntfile.js

```diff
@@ -0,0 +1,37 @@
+module.exports = function(grunt) {
+    var bowerPrefix = 'bower_components/bower_components/'
+
+    var vendorB2 = [
+        bowerPrefix + 'jquery-1.7.1-legacy/jquery.js',
+        bowerPrefix + 'knockout-2.3.0-legacy/knockout.js',
+        bowerPrefix + 'underscore-legacy/underscore-min.js',
+    ];
+    var vendorB3 = [
+        bowerPrefix + 'jquery/dist/jquery.min.js',
+        bowerPrefix + 'jquery-form/jquery.form.js',
+        bowerPrefix + 'underscore/underscore.js',
+        bowerPrefix + 'jquery.cookie/jquery.cookie.js',
+    ];
+
+    grunt.initConfig({
+        jasmine: {
+            app_manager: {
+                src: 'corehq/apps/app_manager/static/app_manager/js/form_workflow.js',
+                options: {
+                    specs: 'corehq/apps/app_manager/spec/*_spec.js',
+                    vender: vendorB2
+                }
+            },
+        },
+        watch: {
+            app_manager: {
```

Comment:
this is also kind of nice. you can set `grunt watch:app_manager` and it will rebuild the `_SpecRunner.html` whenever you make a file change